support/support.rotation

```diff
@@ -5,28 +5,14 @@
 #@ slack: #eventing-questions
 #@ slacklink: https://knative.slack.com/archives/C017X0PFC0P
 
-
-2021-11-08T01:00:00Z | benmoss
-2021-11-15T01:00:00Z | devguyio
-2021-11-22T01:00:00Z | aliok
-2021-11-29T01:00:00Z | antoineco
-2021-12-06T01:00:00Z | aslom
-2021-12-13T01:00:00Z | matzew
-2021-12-20T01:00:00Z | N/A - break
-2021-12-27T01:00:00Z | N/A - break
-2022-01-03T01:00:00Z | odacremolbap
-2022-01-10T01:00:00Z | lberk
-2022-01-17T01:00:00Z | pierDipi
-2022-01-24T01:00:00Z | lionelvillard
-2022-01-31T01:00:00Z | gabo1208
-2022-02-07T01:00:00Z | benmoss
-2022-02-14T01:00:00Z | devguyio
-2022-02-21T01:00:00Z | aliok
-2022-02-28T01:00:00Z | antoineco
-2022-03-07T01:00:00Z | aslom
-2022-03-14T01:00:00Z | matzew
-2022-03-21T01:00:00Z | odacremolbap
-2022-03-28T01:00:00Z | lberk
-2022-04-04T01:00:00Z | pierDipi
-2022-04-11T01:00:00Z | lionelvillard
-2022-04-18T01:00:00Z | gabo1208
+2022-04-25T01:00:00Z | xtreme-sameer-vohra
```

Comment:
@xtreme-sameer-vohra, this is already the current week, if you aren't available for the rotation I can cover you.

Reply:
Hey @pierDipi 
I missed this so thank you for covering for me.
Can you please remove me from the rotation for the future as I won't be able to dedicate the time to do it justice.